external/os-autoinst-common/README.md

```diff
@@ -8,7 +8,7 @@ This repository is to be used as a
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/git-subrepo.svg)](https://repology.org/project/git-subrepo/versions)
 
-and in `devel:openQA`, e.g. https://download.opensuse.org/repositories/devel:/openQA/openSUSE_Leap_15.1
+and in `devel:openQA`, e.g. https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2/
```

Comment:
but devel:openQA is not the same as devel:openQA:Leap:15.2 . The latter is correct but then I guess the first should say `devel:openQA:Leap:15.2` as well, right?

Reply:
Regardless what the answer to your question is: This change needed to go to os-autoinst-common first and then pulled in again. You can ask the creator of the [original PR](https://github.com/os-autoinst/os-autoinst-common/pull/2) what he was thinking at the time :-)

Reply:
Likely the answer is always "not much thinking" :D ok, this is just an update from an external repo. I am still struggling with this sub repo approach when it also causes confusions like this :D